docs/learn-treasury.md

```diff
@@ -2,24 +2,72 @@
 
-When a stakeholder wishes to propose a spend from the treasury, they must reserve a deposit totaling 5% of the proposed spend. This deposit will be slashed if the proposal is rejected, and returned if the proposal was accepted.
+When a stakeholder wishes to propose a spend from the treasury, they must reserve a deposit totaling 5% of the proposed spend (see below for variations). This deposit will be slashed if the proposal is rejected, and returned if the proposal was accepted.
 
 Proposals may consist of (but are not limited to):
 
- - Infrastructure deployment and continued operation.
- - Network security operations (monitoring services, continuous auditing).
- - Ecosystem provisions (collaborations with friendly chains).
- - Marketing activities (advertising, paid features, collaborations).
- - Community events and outreach (meetups, pizza parties, hackerspaces).
- - Software development (wallets and wallet integration, clients and client upgrades).
+- Infrastructure deployment and continued operation.
+- Network security operations (monitoring services, continuous auditing).
+- Ecosystem provisions (collaborations with friendly chains).
+- Marketing activities (advertising, paid features, collaborations).
+- Community events and outreach (meetups, pizza parties, hackerspaces).
+- Software development (wallets and wallet integration, clients and client upgrades).
 
-The treasury is ultimately controlled by the stakeholders, and how the funds will be spent is up to their judgment.
+The treasury is ultimately controlled by the council, and how the funds will be spent is up to their judgment.
 
-## Resources
+## Funding the Treasury
+
+The Treasury is funded from different sources:
+
+1. When a validator is slashed for equivocation or invalid attestations of parachain blocks, the slashed amount is sent to the Treasury with a reward going to the entity which reported the validator (another validator or fisherman). The reward is taken from the slash amount and varies per offence and number of reporters.
+2. Transaction fees paid on the Relay Chain for executing balance transfers, governance votes, council actions or any other function allowed by [pallets](https://substrate.dev/docs/en/tutorials/adding-a-module-to-your-runtime/) included in the runtime.
```

Comment:
This information about pallets makes it more confusing and is an implementation detail. Polkadot's runtime construction has nothing to do with transaction fees in this context.
```suggestion
2. Transaction fees: A portion of each block's transaction fees goes to the Treasury, with the remainder going to the block author.
```